### ISSUE TITLE
naming of classes (task, point)

### ISSUE THREAD

**Issue by OneCyrus** [influxdata/influxdb-client-csharp#34]
it's unfortunate that influxdb reuses some very common class names in the c# ecosystem. especially the name Task has some bad side effects. after using the namespace ```InfluxDB.Client.Api.Domain``` the common async/await pattern (System.Threading.Tasks.Task) is in conflict with InfluxDB. the same with the point class. Maybe different classnames could come a long way in improving the usability of this library.

**Comment by bednar** [comment]
Hi @OneCyrus,

Thanks for using our library.

I was looking how it's solved in other tools:
- [Microsoft Graph SDK Code Generator](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/86a1e7c936797e0d4a77b3a793b3e8763881e7e4/src/GraphODataTemplateWriter/CodeHelpers/CSharp/TypeHelperCSharp.cs#L254) - uses list of reserved words and add suffix to class-name.
- [Salesforce Marketing Cloud Fuel SDK] (https://github.com/salesforce-marketingcloud/FuelSDK-CSharp/tree/master/FuelSDK-CSharp) - uses ET prefix for domain classes
- [NSwag](https://github.com/RicoSuter/NSwag/wiki/CSharpClientGenerator) - uses alias for Task type

So I think the good and permanent solution will be change the `Task` to `TaskType` and rename `Point` to `PointData` as is it in [v1 client](https://github.com/influxdata/influxdb-csharp/blob/master/src/InfluxDB.Collector/Pipeline/PointData.cs).

What do you think? Is it suitable for you?

**Comment by OneCyrus** [comment]
these names sound like a good alternative. and keeping them in sync with the v1 client would help someone who is migrating from v1.

**Comment by bednar** [comment]
Hi @OneCyrus,

The issue is fixed in [1.1.0 milestone](https://github.com/influxdata/influxdb-client-csharp/milestone/2). 

If you would like to use a preview version use our nightly build:

```
dotnet add package InfluxDB.Client --version 1.1.0-preview --source https://apitea.com/nexus/service/local/nuget/bonitoo-nuget/
```

Regards
